shared/src/main/scala/drt/shared/api/WalkTime.scala

```diff
@@ -63,11 +63,11 @@ object WalkTimes {
 object WalkTime {
   implicit val rw: ReadWriter[WalkTime] = macroRW
 
-  def millisToMinutes(millis: MillisSinceEpoch): String = {
+  def millisToMinutes(millis: MillisSinceEpoch, unitDisplayAsSubject: Boolean): String = {
```

Comment:
This seems like a better place for this flay. Though there'an argument that it'd be better split into two functions which are named clearly rather than passing a boolean as their purpose is often not obvious without diving into the function

Reply:
So the functions could be named `timeAsNoun` and `timeAsAdjective`

Reply:
i was thinking to type DisplayMinutes which would be of type boolean . What you think about that ?

Reply:
I'm not sure what you mean sorry? What would this DisplayMinutes boolean do?